test/module/irohad/consensus/yac/CMakeLists.txt

```diff
@@ -20,38 +20,45 @@ addtest(cluster_order_test cluster_order_test.cpp)
 target_link_libraries(cluster_order_test
     yac
     shared_model_cryptography
+    shared_model_proto_backend
```

Comment:
Move to `irohad/consensus/yac/CMakeLists.txt` for `target_link_libraries(yac`, because backend (e.g. `SignatureBuilder`) is used in `yac_pb_converters.hpp`.

Linking from tests below can be also removed after this, because they are already linked with `yac`.